src/lib/eliom_service_sigs.shared.mli

```diff
@@ -129,60 +133,57 @@ module type S = sig
       the list will be URL-encoded.
 
       The parameter labelled [~prefix] contains all what you want to
-      put before the path. It usually starts with "http://" plus the
+      put before the path. It usually starets with "http://" plus the
       name of the server. The prefix is not URL encoded.
 
       The whole URL is constructed from the prefix, the path and GET
       parameters. Hence, an empty prefix can be used to make a link to
       another site of the same server.
 
       See {!val:service} for a description of the optional
-      [~keep_nl_params] and [~rt] parameters.  *)
-  val external_service :
+      [~keep_nl_params] and [~ret] parameters.  *)
+  val create_external :
```

Comment:
Just another case operationally speaking, but with additional type parameters. It would add an `ext` type parameter to `Id.t`, and I think a `reg` type also.